learning_journal/learning_journal/templates/detail.jinja2

```diff
@@ -1,9 +1,9 @@
 {% extends "index.jinja2" %}
 {% block body %}
   <article>
-    <h2 class="entrytitle">{{ title }}</h2>
-      <p class="timestamp">{{ creation_date }} </p>
-      <p class="entry">{{ body }}</p>
-      <p class="editentry"><a class="btn btn-primary" href="{{ request.route_url("update", id=id) }}">Edit</a></p>
+    <h2 class="entrytitle">{{ entry.title }}</h2>
+      <p class="timestamp">{{ entry.creation_date }} </p>
+      <p class="entry">{{ entry.body }}</p>
```

Comment:
consider a future where your entries are themselves written in HTML.  Would you want to try to shoehorn all of a single entry into one paragraph tag?  YOu might consider using a `<div>` tag as the container for your entry body text instead.  This would allow you to have a block-level tag that didn't have a semantic meaning like `<p>` does.